File: Bit Algorithm/Find Non-Repeated Element/SolutionByTarun.py
def firstNR(arr, n): 
  
	for i in range(n): 
		j = 0
		while(j < n): 
			if (i != j and arr[i] == arr[j]): 
				break
			j += 1
		if (j == n): 
			return arr[i] 
	
	return -1
	
# Driver code
n=input('Enter size : ')
arr=[]
for i in range(0,n):
  arr.append(input('enter element'))
print(firstNR(arr, n))

File: Data Structure/Array Or Vector/Search An Element in Unsorted Array/solutionbyTarun.cpp
#include <iostream>
using namespace std;

int main()
{
    int element, size;
    cin >> size;
    int arr[size];
    for (int i = 0; i < size; i++) {
        cin >> arr[i];
    }
    cin >> element;

    for (int i = 0; i < size; i++) {
        if (arr[i] == element) {
            cout << "YES" << endl;
            return 0;
        }
    }
    cout << "NO" << endl;
    return 0;
}
